pkg/cloudprovider/providers/openstack/openstack_test.go

```diff
@@ -33,7 +33,7 @@ import (
 	"github.com/gophercloud/gophercloud/openstack/compute/v2/servers"
 	"github.com/spf13/pflag"
 
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
I don't understand the change here

Reply:
Its automatically added I suppose , removed now.`